test/views/authenticated/ChangeWhichTaxYearsViewSpec.scala

```diff
@@ -89,7 +82,7 @@ class ChangeWhichTaxYearsViewSpec extends CheckboxViewBehaviours[TaxYearSelectio
           (option, i) <- taxYearsAndAmounts.zipWithIndex
         } yield {
           val id = form("value")(s"[$i]").id
-          doc.select(s"label[for=$id]").text mustEqual option._1.message.html.toString
+          doc.select(s"label[for=$id]").text contains option._1.message.html.toString
```

Comment:
`doc.select(s"label[for=$id]").text mustBe s"${option._1.message.html.toString} ${messages("changeWhichTaxYears.columnHeading2")} £${option._2}"`